### ISSUE TITLE
mto: unable to pronounce 20s (21, 22, etc.)

### ISSUE THREAD

**Issue by codeofdusk** [espeak-ng/espeak-ng#1230]
STR:
Attempt to read the numbers 21, 22, 23... 29 with mto espeak-ng.

Expected: Numbers are read.

Actual: eSpeak remains silent.

Version tested: commit dc153592.

**Comment by pvagner** [comment]
Hello,
With commit e322584a147fe2666bad2c9e3f69e5ef88355ed5
```espeak -v mto "21"```
is working fine for me. However I see different issue I've just filled #1302

**Comment by sthibaul** [comment]
Most probably fixed by https://github.com/espeak-ng/espeak-ng/commit/e322584a147fe2666bad2c9e3f69e5ef88355ed5 indeed, thanks @pvagner

**Comment by codeofdusk** [comment]
Now 21, 22, 23... are read identically to 1, 2, 3.

**Comment by sthibaul** [comment]
@codeofdusk really? I'm hearing `j@Rm'A` at the beginning of 21, 22, 23, etc.

**Comment by valdisvi** [comment]
General espeak-ng problem is that it doesn't fall back to reading single digits for composite numbers. Actual problem is that rules for Totontepec Mixe language has defined pronunciation for single digits, but not for tens (hundreds etc):
```
espeak -v mto -X "2 1"
Found: '_2' [mahtsk#]  
Found: '_1' [toU?k#]  
'mahtsk# t'oU?k#
espeak -v mto -X "21"
Found: '_1' [toU?k#]  
t'oU?k#
```
For comparison pronunciation for Latvian: 
espeak -v lv -X "2 1"
Found: '_2' [d'ivi]  
Found: '_1' [v'iens]  
d'ivi v'iens
valdis@oto:~/es(master)$ espeak -v lv -X "21"
Found: '_2X' [d'ivdesmit||]  
Found: '_1' [v'iens]  
d'ivdesmit v'iens
```

**Comment by valdisvi** [comment]
There was typo in rules, which I fixed in 96ab4ec3c29971e2864e9a18818b77ed9ca4c59c

**Comment by sthibaul** [comment]
> Hello, With commit [e322584](https://github.com/espeak-ng/espeak-ng/commit/e322584a147fe2666bad2c9e3f69e5ef88355ed5) `espeak -v mto "21"` is working fine for me. However I see different issue I've just filled #1302

I was confused by your message actually: e322584 was't about mto, but nog.

Anyway, thanks @valdisvi !
